Modules/Devices/test/Devices.Application.Tests/Tests/Tiers/Commands/DeleteTier/HandlerTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Application.IntegrationEvents.Outgoing;
+using Backbone.Modules.Devices.Application.Tests.Extensions;
+using Backbone.Modules.Devices.Application.Tiers.Commands.DeleteTier;
+using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using Backbone.Modules.Devices.Domain.Entities;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using ApplicationException = Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Tiers.Commands.DeleteTier;
+public class HandlerTests
+{
+    private readonly ITiersRepository _tierRepository;
+    private readonly IEventBus _eventBus;
+    private readonly Handler _handler;
+
+    public HandlerTests()
+    {
+        _tierRepository = A.Fake<ITiersRepository>();
+        _eventBus = A.Fake<IEventBus>();
+        _handler = CreateHandler();
+    }
+
+    private Handler CreateHandler()
+    {
+        var logger = A.Fake<ILogger<Handler>>();
+        return new Handler(_tierRepository, logger, _eventBus);
+    }
+
+    [Fact]
+    public async Task Cannot_Delete_Basic_Tier()
+    {
+        // Arrange
+        var basicTierName = TierName.Create("basic-tier").Value;
```

Comment:
Please inline all simple variables in all tests of this class that are only used once throughout the tests, like the `basicTierName`